letterboxdpy/movie.py

```diff
@@ -209,6 +211,18 @@ def movie_popular_reviews(self, dom) -> dict:
 
             self.popular_reviews.append(curr)
 
+    def movie_title_id(self, script) -> str:
+        try:
+            self.movie_title_id = script['url'].split('/')[4]
+        except Exception:
+            self.movie_title_id = ''
```

Comment:
```py
  def movie_title_id(self, script) -> str:
     try:
         self.movie_title_id = script['url'].split('/')[4]
     except Exception:
         self.movie_title_id = ''
```           
Let's remove this method as well since we already obtain the movie-title-id with the slug.